templates/kafka-pv.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ .Values.global.persistence.kafka.name }}
+  {{- if eq .Values.hosting "aws" }}
+  labels:
+    type: amazonEBS
+  {{- end }}
+spec:
+  accessModes:
+  - ReadWriteMany
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: {{ .Values.global.persistence.kafka.name }}
+  capacity:
+    storage: {{ .Values.global.persistence.kafka.storage }}
+  {{- if eq .Values.hosting "local" }}
+  hostPath:
+    # Directory location on host
+    path: {{ .Values.global.persistence.kafka.hostPath }}
+    type: DirectoryOrCreate
```

Comment:
Is this definitely required. Am sure in another PR @dev-958 removed this field in a different PV as it wasn't required